### DISCUSSION TITLE
Issue integrating leva in a project (R3F:  Svg....)

### DISCUSSION THREAD

**Discussion by aymanhab** [pmndrs/leva#517] [Q&A]
Hello,
I'm trying to incorporate leva in a react based project that includes react three fiber and drei. The important lines in package.json are:
    "@react-three/drei": "^9.57.0",
    "@react-three/fiber": "^8.11.5",
    "three": "^0.151.0",
    "leva": "^0.9.35",

When I import leva, I get the error in run time
R3F: Svg is not part of the THREE namespace

This is just including <Leva collapsed /> in my jsx
There're a bunch other packages included but it's not clear what I'm doing wrong or how to fix this. Any help or pointers are appreciated.

**Comment by aymanhab** [comment]
This seems to be an issue with R3F play with svg rather than leva even though it shows only when trying to integrate leva for some reason, so I'm investigating further on that front but leva appears relatively unrelated.

**Comment by Cosmitar** [comment]
probably your are trying to render `<Leva>` inside the canvas from R3F and Leva is HTML content. You can try Drei's Html component:
```jsx
import { Html } from '@react-three/drei'
import { Leva } from 'leva'

const MyR3FComponent = () => <Html><Leva /></Html>
```

or any other approach to render html outside the canvas
